crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -224,7 +224,7 @@ where
                 if scope_index != GLOBAL_SCOPE_INDEX {
                     let context = self.ctx.execution_context();
                     let scope = &mut self.ctx.scopes[scope_index];
-                    let usage = Some((scope.id, Range::from_located(stmt)));
+                    let usage = Some((scope.id, stmt.into()));
```

Comment:
I prefer to use `Range::from(stmt)` over `stmt.into()` because the former is explicit about the conversation (I need to use IDE features to know what conversation `stmt.into()` performs).